src/tree/RevisionItem.ts

```diff
@@ -44,7 +44,7 @@ export class RevisionItem implements RevisionsItemModel {
             label: this.revision.name,
             iconPath: this.iconPath,
             description,
-            contextValue: `${this.revision.active ? 'active' : 'inactive'};revision`,
```

Comment:
value passed to `createContextValue` should be like `['active', 'revision']`. The util adds the `;` too.